model-api-gen/src/main/kotlin/org/modelix/metamodel/generator/TypescriptMMGenerator.kt

```diff
@@ -8,7 +8,7 @@ import java.nio.file.Files
 import java.nio.file.Path
 import kotlin.io.path.writeText
 
-class TypescriptMMGenerator(val outputDir: Path, val nameConfig: NameConfig = NameConfig()) {
+class TypescriptMMGenerator(val outputDir: Path, val nameConfig: NameConfig = NameConfig(), private val includeTBarrels: Boolean = false) {
```

Comment:
Typo. `includeTBarrels` should be `includeTSBarrels`

Reply:
Fixed.